content/spin/spin-oci.md

```diff
@@ -60,11 +69,11 @@ $ cd spin-react-fullstack
 $ spin build
 ```
 
-Now we're ready to push the application. Run the `spin oci push` command (for Spin v0.8.0) or the `spin registry push` command (for Spin v0.9.0 onwards); to push your application to the registry: 
+Now we're ready to push the application. Run the `spin registry push` command to push your application to the registry: 
 
 {{ tabs "spin-version" }}
 
-{{ startTab "v0.10.0"}}
+{{ startTab "v1.1.0"}}
```

Comment:
Suggest dropping the tab block or it implies the page doesn't cover v1.0.

Reply:
That's a great suggestion; and then if in the future versions beyond the current version have syntax changes we can introduce the versioned code blocks back in again (stating the appropriate version numbers)

Reply:
Changes coming in new push to this branch ...